addons/website/tests/test_res_users.py

```diff
@@ -47,14 +49,14 @@ def test_change_login_no_website(self):
             user_belle.login = 'Pou'
 
     def test_same_website_message(self):
+        self.env.registry.enter_test_mode(self.env.cr)
+        self.addCleanup(self.env.registry.leave_test_mode)
 
-        @check # Check decorator, otherwise translation is not applied
-        def check_new_test_user(dbname):
-            new_test_user(self.env(context={'land': 'en_US'}), login='Pou', website_id=self.website_1.id)
+        env = self.env(context={'lang': 'en_US'}, cr=self.env.registry.cursor())
+        new_test_user(env, login='Pou', website_id=self.website_1.id)
```

Comment:
Please explain why you use a test cursor here.
Is it because `retrying` attempts to `commit`?

Reply:
There is a missing comment indeed